client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,20 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
+export function makeLoggedOutLayout( context, next ) {
+	const { store, primary, secondary, tertiary } = context;
+	context.layout = (
+		<ReduxProvider store={ store }>
+			<LayoutLoggedOut primary={ primary }
+				secondary={ secondary }
+				tertiary={ tertiary } />
+		</ReduxProvider>
+	);
+	next();
+};
```

Comment:
Should this function `delete` primary, secondary, tertiary? Likely an irrelevant question _right now_, but later on there could be a point to avoiding confusion (if some middleware sees `.primary` it may assume the wrong things). Just something to keep in mind.